### ISSUE TITLE
#Couldn't remove temporary file#

### ISSUE THREAD

**Issue by Catherine9826** [DecodeGenetics/Ratatosk#41]
Thank you for the amazing tool. I met a problem when I used Ratatosk.
   This is my command: Ratatosk correct -v -c 16 -s transcripts.stringtie.fa -l Tam.transcripts.fa -o out_long_reads
   I got the file ‘out_long_reads‘, but it said 'Couldn't remove temporary file' .Does it matter?

**Comment by GuillaumeHolley** [comment]
Hi @Catherine9826,

Thanks for reporting this issue. It does not matter for your output, it is just a warning about whether temporary data written by Ratatosk to disk but not needed anymore were successfully removed. Turns out I programmed this incorrectly and the warning message is printed if the data are successfully removed from disk :) Either way, output data is not affected by this.

I'll keep the issue open to remember to change this for the nest release.

Guillaume
